packages/web/src/app.tsx

```diff
@@ -24,7 +26,9 @@ import './index.css'
 const AudiusApp = () => {
   return (
     <Provider store={store}>
-      <AudiusQueryContext.Provider value={{ apiClient }}>
+      <AudiusQueryContext.Provider
```

Comment:
love this, as it gets bigger, we can maybe add a provider in a separate file that does all the config